lesson-9-maximum-slice-problem/maxslicesum.py

```diff
@@ -0,0 +1,7 @@
+def solution(A):
+    max_ending = max_slice = -1000000
+    for a in A:
```

Comment:
well, idk, it seems appropriate though. seeing that a is an element of A... it doesn't get used much either way